files/build_templates/docker_image_ctl.j2

```diff
@@ -7,18 +7,32 @@ HWSKU=`sonic-cfggen -m /etc/sonic/minigraph.xml -v minigraph_hwsku`
 start() {
     docker inspect --type container {{docker_container_name}} &>/dev/null
     if [ "$?" -eq "0" ]; then
+{%- if docker_container_name == "database" %}
```

Comment:
Is is necessary to distinguish the database container here? 
I understand database container might be agnostic to HWSKU, but the container would have a wrong hwsku path inside.  On the other hand, if we want to save some time for bootup, there are more containers that are HWSKU agnostic.

Reply:
You are absolutely right.

The mounting will be removed by Taoyu. His change is already in review. So I didn't duplicate the change. He will have to do a merge after my change went in.

Reply:
with or not to moun hswku into database docker is a separate issue, that will be addressed by taoyu. I get that. But, for you change, I think you do not need to treat database docker differently with or ot the hwsku is mounted. You do not need to add database specific logic here.

Reply:
I chatted with Taoyu.

Basically, once we move to config_db, we won't be able to get HWSKU at the time when database container is starting.

If I include Taoyu's logic of not getting HWSKU and not mounting HWSKU into database container, will you be ok with this change?

Reply:
Can you take a look at the latest change to see if you like it or not?

With this change, I included Taoyu's logic, not mounting hwsku in database container.

Thanks,
Ying